src/lib/pmd/PmdSupport.ts

```diff
@@ -28,15 +28,6 @@ export abstract class PmdSupport extends CommandLineSupport {
 		return classpathEntries;
 	}
 
-	protected isSuccessfulExitCode(code: number): boolean {
```

Comment:
`pmd-cataloger`'s only successful exit code is 0, so this method shouldn't have been here.